spec/components/bali/calendar_spec.rb

```diff
@@ -0,0 +1,238 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Bali::Calendar::Component, type: :component do
+  before do
+    @options = {}
+  end
+
+  let(:component) { Bali::Calendar::Component.new(**@options) }
+  let(:monday) { Date.current.prev_occurring(:monday) }
+  let(:friday) { Date.current.prev_occurring(:friday) }
+
+  it 'renders calendar component with all week' do
+    @options.merge!(start_date: '2020-01-01', all_week: true)
+    render_inline(component) do |c|
+      c.header(period: c.period, start_date: '2020-01-01')
+    end
+
+    expect(rendered_component).to have_css '.calendar-component'
+    expect(rendered_component).to have_css '.month-view'
+    expect(rendered_component).to have_css 'tr > th.has-text-centered', text: 'Monday'
+    expect(rendered_component).to have_css 'tr > th.has-text-centered', text: 'Friday'
+    expect(rendered_component).to have_css 'tr > th.has-text-centered', text: 'Saturday'
+    expect(rendered_component).to have_css 'tr > th.has-text-centered', text: 'Sunday'
+    expect(rendered_component).to have_css '.header > .columns > .column > h3.title',
+                                           text: 'January 2020'
```

Comment:
```suggestion
    expect(rendered_component).to have_css(
      'tr > th.has-text-centered', text: 'Monday',
      'tr > th.has-text-centered', text: 'Friday',
      'tr > th.has-text-centered', text: 'Saturday',
      'tr > th.has-text-centered', text: 'Sunday')
```

Reply:
I cannot use this syntaxis because I use CSS with text, for use this syntaxis I need to use only CSS